### ISSUE TITLE
[🐛BUG] bug in Pretrain in the NeuMF model

### ISSUE THREAD

**Issue by irecsys** [RUCAIBox/RecBole#1097]
def load_pretrain(self):
        r"""A simple implementation of loading pretrained parameters.

        """
        mf = torch.load(self.mf_pretrain_path)
        mlp = torch.load(self.mlp_pretrain_path)
        self.user_mf_embedding.weight.data.copy_(mf.user_mf_embedding.weight)
        self.item_mf_embedding.weight.data.copy_(mf.item_mf_embedding.weight)
        self.user_mlp_embedding.weight.data.copy_(mlp.user_mlp_embedding.weight)
        self.item_mlp_embedding.weight.data.copy_(mlp.item_mlp_embedding.weight)

        for (m1, m2) in zip(self.mlp_layers.mlp_layers, mlp.mlp_layers.mlp_layers):
            if isinstance(m1, nn.Linear) and isinstance(m2, nn.Linear):
                m1.weight.data.copy_(m2.weight)
                m1.bias.data.copy_(m2.bias)

        predict_weight = torch.cat([mf.predict_layer.weight, mlp.predict_layer.weight], dim=1)
        predict_bias = mf.predict_layer.bias + mlp.predict_layer.bias

        self.predict_layer.weight.data.copy_(0.5 * predict_weight)
        self.predict_layer.weight.data.copy_(0.5 * predict_bias)

1. Bug report
self.predict_layer.weight.data.copy_(0.5 * predict_bias)
should be
self.predict_layer.bias.data.copy_(0.5 * predict_bias)

2. Question
why you multiple 0.5 here

**Comment by chenyuwuxin** [comment]
@irecsys Hello,thanks for your attention to RecBole!
In NeuMF, we load the pretrained mf and mlp which are models that only train one side of NeuMF. In the final prediction stage, we hope to combine the prediction results of both sides. That is, we want the final rating to be the average of rating on both sides. So the parameters are multiplied by 0.5.

**Comment by 2017pxy** [comment]
@irecsys Thanks for your bug report and we have already fixed the bug in #1186
